main.tf

```diff
@@ -44,92 +52,104 @@ resource "aws_db_subnet_group" "this" {
   tags = var.tags
 }
 
+################################################################################
+# Cluster
+################################################################################
+
 resource "aws_rds_cluster" "this" {
   count = local.create_cluster ? 1 : 0
 
-  # Notes:
-  # iam_roles has been removed from this resource and instead will be used with aws_rds_cluster_role_association below to avoid conflicts per docs
-
-  global_cluster_identifier      = var.global_cluster_identifier
-  enable_global_write_forwarding = var.enable_global_write_forwarding
-  cluster_identifier             = var.name
-  replication_source_identifier  = var.replication_source_identifier
-  source_region                  = var.source_region
-
-  engine                              = var.engine
-  engine_mode                         = var.engine_mode
-  engine_version                      = local.is_serverless ? null : var.engine_version
```

Comment:
this fixes the serverless v1 upgrade so users can now set the engine version - not sure when this changed but now its configurable for #313